imports/plugins/core/ui/client/components/cards/settingsCard.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Settings Card is a composite component to standardize the
+ * creation settings cards (panels) in the dashboard.
+ */
+
+import React, { Component, PropTypes } from "react";
+import Blaze from "meteor/gadicc:blaze-react-component";
+import { Card, CardHeader, CardBody } from "/imports/plugins/core/ui/client/components";
+
+class SettingsCard extends Component {
```

Comment:
This most probably is RC code here. We can update this PR if you think it fits RC's requirements from the description and screenshots. It's just a cleaner version of the shop > shop settings, and without the additional owner stuff.